apps/design/backend/src/store.ts

```diff
@@ -3,34 +3,49 @@ import {
   Optional,
   assert,
   find,
-  groupBy,
   iter,
-  throwIllegalValue,
-  unique,
 } from '@votingworks/basics';
 import { Client as DbClient } from '@votingworks/db';
 import {
   DEFAULT_LAYOUT_OPTIONS,
   LayoutOptions,
-  NhCustomContent,
   NhCustomContentByBallotStyle,
 } from '@votingworks/hmpb-layout';
 import {
   Id,
   Iso8601Timestamp,
   Election,
-  DistrictId,
-  PrecinctId,
-  BallotStyleId,
   DEFAULT_SYSTEM_SETTINGS,
   SystemSettings,
   safeParseSystemSettings,
-  CandidateContest,
-  PartyId,
   LanguageCode,
 } from '@votingworks/types';
 import { join } from 'path';
 import { v4 as uuid } from 'uuid';
+import {
+  BooleanEnvironmentVariableName,
+  isFeatureFlagEnabled,
+} from '@votingworks/utils';
+import {
+  BallotLanguageConfig,
+  BallotLanguageConfigs,
+  BallotStyle,
+  Precinct,
+  convertToVxfBallotStyle,
+  hasSplits,
+} from './types';
+import { generateBallotStyles } from './ballot_styles';
+
+export function getTempBallotLanguageConfigsForCert(): BallotLanguageConfigs {
+  const translationsEnabled = isFeatureFlagEnabled(
+    BooleanEnvironmentVariableName.ENABLE_CLOUD_TRANSLATION_AND_SPEECH_SYNTHESIS
+  );
+  return translationsEnabled
+    ? Object.values(LanguageCode).map(
+        (l): BallotLanguageConfig => ({ languages: [l] })
```

Comment:
If I'm remembering right, this is an array because, for multi-language HMPBs, we'll have two languages on the ballot at once, e.g., English and Spanish.

These single-language configs are then for the BMD, is that correct?

Once we build support for multi-language HMPBs, will the ballot language configs include both `[English, Spanish]` (HMPB) and `[Spanish]` (BMD)?

All seems fine! Just checking my understanding

Reply:
That's right, this is allowing for both single-language and multi-language HMPBs - the current assumption is that only the BMD would use non-English single-language ballot styles, but I could see a world where we move to only specifying one language in the ballot style definition and always treating that as multi-language (English + specified language) when generating both the HMPBs and BMDPBs.

Reply:
Perfect, thanks for confirming!